src/openms/source/DATASTRUCTURES/Compomer.cpp

```diff
@@ -256,10 +258,10 @@ namespace OpenMS
       throw Exception::InvalidValue(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION, "Compomer::isSimpleAdduct() does not support this value for 'side'!", String(side));
 
     if (cmp_[side].size() != 1)
-      return false;
+      { return false; }
```

Comment:
to be consistent you should do
```suggestion
    { 
      return false; 
    }
```

Reply:
also below in other files